### ISSUE TITLE
Mapping Error MB

### ISSUE THREAD

**Issue by MrDark-X** [d3vilh/openvpn-ui#62]
Im receiving this "Mapping Error MB" Under Clients Section
![image](https://github.com/d3vilh/openvpn-ui/assets/70789856/a0e00e5e-7057-4d87-8692-bd3e042961c1)

**Comment by d3vilh** [comment]
Openvpn-UI takes this information from[ OpenVPN server management console](https://openvpn.net/community-resources/management-interface/), which should be enabled (first line `management 0.0.0.0 2080` in your `server.conf`), up and running.

If you running 2 containers (openvpn server and openvpn ui) - both containers should use the same Docker subnet and management console must be available from openvpn-ui container (by default it takes container name `openvpn` to address  management console.

What you have to do for fix:
* Confirm you have management console enabled on port 2080 in `server.conf`.
* Confirm your openvpn service/container is up and running (`docker ps | grep openvpn`).
* Confirm you have access to management console from host server (with netcat):
```bash
yejik@superserv:~ $ nc -zv $(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' openvpn) 2080
Connection to 172.18.0.3 2080 port [tcp/*] succeeded!
yejik@ superserv:~ $
```
* Confirm you have access to management console from the openvpn-ui container:
```bash
yejik@ superserv:~ $ docker exec openvpn-ui sh -c "nc -zv $(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' openvpn) 2080"
172.18.0.3 (172.18.0.3:2080) open
yejik@ superserv:~ $
```

if you do have access between containers, than, please share the start logs for both *openvpn* and *openvpn-ui* containers (`docker restart openvpn; docker restart openvpn-ui; docker logs openvpn; docker logs openvpn-ui`)

**Comment by MrDark-X** [comment]
Can u provide a solution for a different scenario?

I have installed the OpenVPN server and OpenVPN-ui on an On-Prem Server 
intel i7 2nd gen 8gb ram and 256gb sata SSD(old-pc)
OS: Ubuntu Server

Here is what I have done so far:
1. I installed the OpenVPN server using [OpenVPN-install-script](https://github.com/angristan/openvpn-install)
2. I have cloned the Openvpn-ui repo and installed it using [standalone-install.sh](https://github.com/d3vilh/openvpn-ui/blob/e2f452d2872a022147f8c58213fa4306f61e65e8/build/standalone-install.sh) I have followed all the steps given in the GitHub and was successfully able to install without any issues
3. When I opened openvpn-ui(web interface) and found this issue

Can u please walkme through to fix this issues

**Comment by d3vilh** [comment]
Then you need to update Management interface address in Openvpn-UI.

Login to UIU, go to `Configuration` > `OpenVPN-UI` and change `openvpn:2080` to `127.0.0.1:2080`.

But before confirm it is available: `nc -zv 127.0.0.1:2080`.

**Comment by akselite** [comment]
127.0.0.1:2080: forward host lookup failed: Unknown host
